ports/esp32/machine_uart.c

```diff
@@ -341,6 +376,11 @@ STATIC const mp_rom_map_elem_t machine_uart_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_readinto), MP_ROM_PTR(&mp_stream_readinto_obj) },
     { MP_ROM_QSTR(MP_QSTR_write), MP_ROM_PTR(&mp_stream_write_obj) },
     { MP_ROM_QSTR(MP_QSTR_sendbreak), MP_ROM_PTR(&machine_uart_sendbreak_obj) },
+
+    { MP_ROM_QSTR(MP_QSTR_INVERSE_TX), MP_ROM_INT(UART_INVERSE_TXD) },
+    { MP_ROM_QSTR(MP_QSTR_INVERSE_RX), MP_ROM_INT(UART_INVERSE_RXD) },
+    { MP_ROM_QSTR(MP_QSTR_INVERSE_RTS), MP_ROM_INT(UART_INVERSE_RTS) },
+    { MP_ROM_QSTR(MP_QSTR_INVERSE_CTS), MP_ROM_INT(UART_INVERSE_CTS) },
```

Comment:
I would suggest to call these constants `INV_TX` etc, just to shorten them to make the code take less space, and scripts using them to take less space.

Reply:
Done